lib/rest-api.php

```diff
@@ -40,6 +40,15 @@ function gutenberg_register_rest_menu_location() {
 }
 add_action( 'rest_api_init', 'gutenberg_register_rest_menu_location' );
 
+/**
+ * Registers the navigation areas REST API routes.
+ */
+function gutenberg_register_rest_navigation_areas() {
+	$nav_menu_location = new WP_REST_Block_Navigation_Areas_Controller();
```

Comment:
$nav_menu_location -> $nav_menu_area

Reply:
I fixed it. Looks like it was copy/pasted from the line above.

Also fixed customizer_nonces below which also copy/pasted!